src/JsonSerializable.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * JsonSerializable interface. This file provides backwards compatibility to PHP 5.3 and ensures
+ * the interface is present in systems where JSON related code was removed.
+ *
+ * @link   http://www.php.net/manual/en/jsonserializable.jsonserialize.php
+ * @since  1.0
+ */
+interface JsonSerializable
```

Comment:
Spróbuj zrobić to na zasadzie https://github.com/maxmind/GeoIP2-php/blob/v2.5.0/src/Compat/JsonSerializable.php lub dodaj `joomla/compat` do zależności. W tym momencie jest to kalka z tego pakietu, z wyciętym copyrightem.

Reply:
joomla-compat odpada, bo wymaga 5.3.10...

Reply:
Jeśli chodzi o kalkę to jest to zrobione identycznie jak w joomla-compat i działa zarówno na php 5.3 i wyższych.